modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="BaselineAdminNetworkPolicy"_{context}"]
+= BaselineAdminNetworkPolicy
+
+`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `BaselineAdminNetworkPolicy` to setup and enforce optional baseline network policies that are overridable by users if need be. Rule actions for BANP are `allow` or `deny`.
+
+BANP allows administrators to specify:
+
+* A `subject` that consists of a set of pods that is selected by the policy.
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+[discrete]
+[id="BaselineAdminNetworkPolicy-example"]
+== BaselineAdminNetworkPolicy example
+
+The `BaselineAdminNetworkPolicy` resource is a cluster singleton `object` that can be used as a guardrail policy incase a passed traffic policy does not match any `NetworkPolicies` in the cluster. You must use `default` as the name when creating a `BaselineAdminNetworkPolicy` object.
```

Comment:
It can also be used if there are no ANPs and NPs at all... so not just for PASS, it can also be just used as is for default guardrails use case (please the main KEP I had linked in slack for more details)